systems/plants/shapes/Geometry.cpp

```diff
@@ -154,6 +234,23 @@ namespace DrakeShapes
     return new Mesh(*this);
   }
 
+  void Mesh::getPoints(Eigen::Matrix3Xd &point_matrix) const
+  {
+      extractMeshVertices(point_matrix);      
+  }
+
+  void Mesh::getBoundingBoxPoints(Matrix3Xd &bbox_points) const
+  {
+      Matrix3Xd mesh_vertices;
+      extractMeshVertices(mesh_vertices);
+
+      Vector3d min_pos = mesh_vertices.rowwise().minCoeff();
+      Vector3d max_pos = mesh_vertices.rowwise().maxCoeff();
+      Vector3d size = max_pos - min_pos;
+      Geometry::getBoundingBoxPoints(size(0)/2, size(1)/2, size(2)/2, bbox_points); 
```

Comment:
What if they're not centered around 0 (i.e. the case (min_pos + max_pos) / 2.0 is not a zero vector)?